### PR TITLE
Update learn-nextflow-in-2022.md

### PR DESCRIPTION
**Pull request by sateeshperi**
add Nextflow Tutorial - Variant Calling Edition link

### CODE REVIEW

File: content/blog/2022/learn-nextflow-in-2022.md [hunk 1]
```diff
@@ -89,19 +89,27 @@ The workshop can be opened on [Gitpod](https://gitpod.io/#https://github.com/car
 
 👉 You can find the course in [The Carpentries incubator](https://carpentries-incubator.github.io/workflows-nextflow/index.html).
 
-### 6. Managing Pipelines in the Cloud - GenomeWeb Webinar
+### 6. Nextflow Tutorial - Variant Calling Edition
+
+The [Nextflow Tutorial - Variant Calling Edition](https://sateeshperi.github.io/nextflow_varcal/nextflow/) has been adapted from the [Nextflow Software Carpentry](https://carpentries-incubator.github.io/workflows-nextflow/index.html) training material above and [Data Carpentry: Wrangling Genomics Lesson](https://datacarpentry.org/wrangling-genomics/). Learners will have the chance to learn along with nextflow basics, nf-core, to convert a variant-calling bash-script into a nextflow workflow and modularize the pipeline using DSL2 modules and sub-workflows.
```
Comment (drpatelh) [inline content/blog/2022/learn-nextflow-in-2022.md:94]:
```suggestion
This [Nextflow Tutorial - Variant Calling Edition](https://sateeshperi.github.io/nextflow_varcal/nextflow/) has been adapted from the [Nextflow Software Carpentry training material](https://carpentries-incubator.github.io/workflows-nextflow/index.html) and [Data Carpentry: Wrangling Genomics Lesson](https://datacarpentry.org/wrangling-genomics/). Learners will have the chance to learn Nextflow and nf-core basics, to convert a variant-calling bash-script into a Nextflow workflow and to modularize the pipeline using DSL2 modules and sub-workflows. 
```

**Review by drpatelh — APPROVED** [review]
Nice! Thanks @sateeshperi

**Comment by drpatelh** [comment]
Ping @evanfloden @pditommaso

**Comment by pditommaso** [comment]
Cool, thanks a lot for contributing this. Just one note, to not change current numbering, what about adding the new paragraph at the bottom, having as title *Bonus: Nextflow Tutorial - Variant Calling Edition* ?

**Comment by sateeshperi** [comment]
yes totally works for me

**Comment by pditommaso** [comment]
Nice, could you please amend the PR ?

**Review by pditommaso — APPROVED** [review]

**Comment by pditommaso** [comment]
Awesome thanks!

**Pull request merged**